pyonepassword/api/authentication.py

```diff
@@ -4,3 +4,13 @@
     EXISTING_AUTH_REQD,
     ExistingAuthEnum
 )
+
+# This causes these types to properly re-exported
+# https://mypy.readthedocs.io/en/stable/config_file.html?highlight=export#confval-implicit_reexport
+# anything that gets imported needs to be added to this list
```

Comment:
`__all__` is also restricting what gets included when you do a `from some_module import *` and that's why putting names of things in __all__ makes flake8 be quiet about those things being unused imports

Reply:
Yes, I've always generally avoided using `__all__` because I don't think wildcard imports should be used generally, but of the solutions available here, it seemed the cleanest. The other option is the redundant `from bar import foo as foo`, which isn't intuitive to look at

Reply:
yeah, wildcard imports are an anti pattern, but using `__all__` to get rid of the unused import lint seems reasonable :)